src/test_driver/happy/tests/standalone/inet/test_inet_multicast_01.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python3
```

Comment:
What does the number 01 mean

"test_inet_multicast_01"

Will the tests have to run in order?

Reply:
Renamed to `test_inet_multicast_five_nodes_on_wifi.py`